app/src/main/java/org/stepik/android/view/lesson/ui/activity/LessonActivity.kt

```diff
@@ -265,4 +271,48 @@ class LessonActivity : FragmentActivityBase(), LessonView, NextMoveable {
         // todo: use discussion id after comments refactor
         screenManager.openComments(this, step.discussionProxy, step.id)
     }
+
+    override fun showRateDialog() {
```

Comment:
Let's use common code for dialogs like in `CourseContentFragment::showExplainPermissionsDialog`